Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1015,6 +1020,97 @@ public void Restart()
 
         protected abstract void RestartInternal();
 
+        /// <summary>
+        /// Comparison function used in <see cref="GetPlugins" />.
+        /// </summary>
+        /// <param name="a">Item to compare.</param>
+        /// <param name="b">Item to compare with.</param>
+        /// <returns>Boolean result of the operation.</returns>
+        private static int VersionCompare(
+            (Version PluginVersion, string Name, string Path) a,
+            (Version PluginVersion, string Name, string Path) b)
+        {
+            int compare = string.Compare(a.Name, b.Name, true, CultureInfo.InvariantCulture);
+
+            if (compare == 0)
+            {
+                return a.PluginVersion.CompareTo(b.PluginVersion);
+            }
+
+            return compare;
+        }
+
+        /// <summary>
+        /// Returns a list of plugins to install.
+        /// </summary>
+        /// <param name="path">Path to check.</param>
+        /// <param name="cleanup">True if an attempt should be made to delete old plugs.</param>
+        /// <returns>Enumerable list of dlls to load.</returns>
+        private IEnumerable<string> GetPlugins(string path, bool cleanup = true)
+        {
+            var dllList = new List<string>();
+            var versions = new List<(Version PluginVersion, string Name, string Path)>();
+            var directories = Directory.EnumerateDirectories(path, "*.*", SearchOption.TopDirectoryOnly);
+
+            foreach (var dir in directories)
+            {
+                try
+                {
+                    var manifest = _jsonSerializer.DeserializeFromFile<PlugInManifest>(Path.Combine(dir, "meta.json"));
```

Comment:
Some plugins might not have a `meta.json` file, so they will now no longer be loaded

Reply:
So what do we do? Load them anyway - or make a breaking change to try and solve the plug in issues once and for all?

Reply:
Create a dummy one in the folder if one isn't included in the package?

Or is it a migration routine - create one from the dll that's installed?

Reply:
As far as I know `meta.json` is only created if the plugin maintainer creates the repo using https://github.com/oddstr13/jellyfin-plugin-repository-manager

I would attempt to load the plugin anyway, at least for now

Reply:
Loads any unnamed as foldername version 0.0.0.1, so that any subsequent with a meta.js will be loaded instead.

eg. Cover Art Archive, 0.0.0.1